tap_shopify/schemas/definitions.json

```diff
@@ -806,6 +869,189 @@
           "null",
           "integer"
         ]
+      },
+      "presentment_title": {
+        "type": [
+          "null",
+          "string"
+        ]
+      },
+      "presentment_variant_title": {
+        "type": [
+          "null",
+          "string"
+        ]
+      },
+      "user_id": {
+        "type": [
+          "null",
+          "integer"
+        ]
+      },
+      "variant_price": {
+        "type": [
+          "null",
+          "string"
+        ],
+        "format": "singer.decimal"
+      },
+      "duties": {},
+      "rank": {
+        "type": [
+          "null",
+          "string"
+        ]
+      },
+      "unit_price_measurement": {
+        "type": [
+          "null",
+          "object"
+        ],
+        "properties": {
+          "measured_type": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "quantity_unit": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "quantity_value": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "reference_unit": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "reference_value": {
+            "type": [
+              "null",
+              "string"
+            ]
+          }
+        }
+      },
+      "selling_plan_application": {
+        "type": [
+          "null",
+          "object"
+        ],
+        "properties": {
+          "selling_plan_id": {
+            "type": [
+              "null",
+              "integer"
+            ]
+          },
+          "api_client_id": {
+            "type": [
+              "null",
+              "integer"
+            ]
+          },
+          "name": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "description": {
+            "type": [
+              "null",
+              "string"
+            ]
+          },
+          "pricing_policies": {
+            "type": [
+              "null",
+              "array"
+            ],
+            "items": {}
```

Comment:
Is there any other data that can be added to the items object? This goes for all the `"items": {}`.

Reply:
This will accept every data. We added this because we cannot find any doc in the Shopify API for the data-type of this field.